src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/PagingQueryString.feature.disabled

```diff
@@ -0,0 +1,106 @@
+Feature: Paging Support for GET requests for Ed-Fi Resources
+
+  
+  @ignore
+  Scenario: Ensure clients can't GET information when filtering using invalid values
+    When sending a GET request to "/ed-fi/schools?limit=<Value>"
+             """
+             """
```

Comment:
Please remove lines 7 and 8. A GET request will never have a body.

Reply:
Complete